HousingResidentInformationAPI/V1/UseCase/ValidatePostcode.cs

```diff
@@ -0,0 +1,17 @@
+using HousingResidentInformationAPI.V1.UseCase.Interfaces;
+using System.Text.RegularExpressions;
+
+namespace HousingResidentInformationAPI.V1.UseCase
+{
+    public class ValidatePostcode : IValidatePostcode
+    {
+        public bool Execute(string postcode)
+        {
+            const string pattern =
+                "^([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|" +
```

Comment:
Is this for the postcode pattern?

Reply:
Yes, this is the regular expression for the postcode. Copied from https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/488478/Bulk_Data_Transfer_-_additional_validation_valid_from_12_November_2015.pdf

Reply:
Cool, thank you